poll/public/handlebars/poll_results.handlebars

```diff
@@ -0,0 +1,43 @@
+<script id="poll-results-template" type="text/html">
+    <h3 class="poll-header">{{display_name}}</h3>
+    <div class="poll-question-container">{{{question}}}</div>
+    <ul class="poll-answers-results poll-results">
+    {{#each tally}}
+        <li class="poll-result">
+            <div class="poll-result-input-container">
+              <input id="answer-{{key}}" type="radio" disabled {{#if choice}}checked="True"{{/if}} />
```

Comment:
```"True"``` is not a valid value for the checked attribute (boolean)
http://www.w3.org/html/wg/drafts/html/CR/infrastructure.html#boolean-attribute

It's either ```checked="checked"``` or simply ```checked``` for checked attributes and omitting the ```checked``` attribute all together if its not checked.